### PR TITLE
DAS-106 : Create Windows Batch scripts for Migrate and Backup tools

### PR DESCRIPTION
**Pull request by swsachith**

### CODE REVIEW

File: features/analytics-core/org.wso2.carbon.analytics.core.server.feature/src/main/resources/bin/analytics-backup.bat [hunk 1]
```diff
@@ -0,0 +1,173 @@
+@echo off
+
+REM ---------------------------------------------------------------------------
+REM        Copyright 2005-2009 WSO2, Inc. http://www.wso2.org
```
Comment (madhawa-gunasekara) [inline features/analytics-core/org.wso2.carbon.analytics.core.server.feature/src/main/resources/bin/analytics-backup.bat]: obsolete copyright header

Comment (swsachith) [inline features/analytics-core/org.wso2.carbon.analytics.core.server.feature/src/main/resources/bin/analytics-backup.bat]: Fixed with the PR : https://github.com/wso2/carbon-analytics/pull/137

File: features/analytics-core/org.wso2.carbon.analytics.core.server.feature/src/main/resources/bin/analytics-backup.bat [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+if "%CARBON_HOME%"=="" set CARBON_HOME=%~sdp0..
+SET curDrive=%cd:~0,1%
+SET wsasDrive=%CARBON_HOME:~0,1%
+if not "%curDrive%" == "%wsasDrive%" %wsasDrive%:
+
+rem find CARBON_HOME if it does not exist due to either an invalid value passed
+rem by the user or the %0 problem on Windows 9x
+if not exist "%CARBON_HOME%\bin\version.txt" goto noServerHome
+
+set AXIS2_HOME=%CARBON_HOME%
```
Comment (sinthuja) [inline features/analytics-core/org.wso2.carbon.analytics.core.server.feature/src/main/resources/bin/analytics-backup.bat]: Remove the unnecessary segments of passing version, debug, etc options, and AXIS_HOME, etc variables.

Comment (swsachith) [inline features/analytics-core/org.wso2.carbon.analytics.core.server.feature/src/main/resources/bin/analytics-backup.bat]: fixed with https://github.com/wso2/carbon-analytics/pull/137

**Pull request merged**